be1-go/hub/standard_hub/hub_state/Peers.go

```diff
@@ -26,10 +27,17 @@ func NewPeers() Peers {
 }
 
 // AddPeerInfo adds a peer's info to the table
-func (p *Peers) AddPeerInfo(socketId string, info method.ServerInfo) {
+func (p *Peers) AddPeerInfo(socketId string, info method.ServerInfo) error {
 	p.Lock()
 	defer p.Unlock()
+
+	_, ok := p.peersInfo[socketId]
+	if ok {
+		return xerrors.Errorf("peersInfo already contains [%s]", socketId)
```

Comment:
I think the logs would be a bit more readable if they included the peer's info than the socket id. Maybe something like "peersInfo already contains socketid *id* for server *peer's address* would be useful for debugging.

Reply:
I changed it to also include the peer's info.